Template.php

```diff
@@ -1500,6 +1500,22 @@ public function expand_by_adsabs() {
         }
       }
       
+      if ($this->wikiname() === 'cite book' || $this->wikiname() === 'citation') { // Possible book and we found book review in journal
+        $book_count = 0;
+        if($this->has('publisher')) $book_count = $book_count + 1;
+        if($this->has('isbn'))      $book_count = $book_count + 2;
+        if($this->has('location'))  $book_count = $book_count + 1;
+        if($this->has('chapter'))   $book_count = $book_count + 2;
+        if($this->has('oclc'))      $book_count = $book_count + 1;
+        if($this->has('lccn'))      $book_count = $book_count + 2;
+        if($this->has('journal'))   $book_count = $book_count - 2;
+        if($this->wikiname() === 'cite book') $book_count = $book_count + 3;
```

Comment:
This format (combined with `$bookcount++` for += 1) might be cleaner and easier to read?
```suggestion
        if($this->wikiname() === 'cite book') $book_count += 3;
```